src/System.Drawing.Common/tests/Imaging/FrameDimensionTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Drawing.Imaging.Tests
+{
+    public class FrameDimensionTests
+    {
+        [Fact]
+        public void Ctor_Guid()
+        {
+            var guid = Guid.NewGuid();
+            FrameDimension fd = new FrameDimension(guid);
+            Assert.Equal(guid, fd.Guid);
+        }
+
+        [Fact]
+        public void DefinedFrameDimensions_ReturnsExpected()
+        {
+            Assert.Equal(new Guid("{6aedbd6d-3fb5-418a-83a6-7f45229dc872}"), FrameDimension.Time.Guid);
```

Comment:
Do you think it's worth turning this into a theory? Since it's limited in length, I don't know